src/commands/lightning/preview/app.ts

```diff
@@ -149,7 +150,10 @@ export default class LightningPreviewApp extends SfCommand<void> {
 
     logger.debug('Determining Local Dev Server url');
     // todo: figure out how to make the port dynamic instead of hard-coded value here
-    const ldpServerUrl = PreviewUtils.generateWebSocketUrlForLocalDevServer(platform, '8081');
+    const ldpServerUrl = PreviewUtils.generateWebSocketUrlForLocalDevServer(
+      platform,
+      `${await LwcDevServerUtils.getLocalDevServerPort()}`
```

Comment:
@maliroteh-sf not sure why the port is consumed as a string?

Reply:
I have this fixed locally so please ignore for now until my next PR.